### DISCUSSION TITLE
1 - Actor not casted on cast from Queerclick / 2 - Series as cast from Fagalicious

### DISCUSSION THREAD

**Discussion by dimmitri9** [CodyBerenson/PGMA-Modernized#249] [Q&A]
This title has an actor in it's cast that is not in the cast of the video itself, from Queerclick

https://www.queerclick.com/qc/2023/05/say-uncle-brother-crush-dalton-riley-and-oliver-beck-rocking-his-boat.php
![Screenshot 2023-06-03 195112](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/3a27b3f6-e62a-4a22-89bd-13129d11c3ad)
![Screenshot 2023-06-03 195134](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/b897bd35-4183-40fb-b95e-987f884dd018)
![Screenshot 2023-06-03 195158](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/b2fbfb53-b68e-4ef9-aa07-db257cba6030)
![Screenshot 2023-06-03 195836](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/20113cc9-5a74-4ba1-81c7-b788b5b7bce4)

--------------------------------------------------------------------

Also, this other title has Rod'sRoom as it's cast, from Fagalicious

![Screenshot 2023-06-03 195938](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/94c43c28-0a5c-40da-8ecf-9b727c2007c6)
![Screenshot 2023-06-03 195956](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/10e63304-028b-4abc-8d8c-f16f0d5a3e44)
![Screenshot 2023-06-03 195947](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/fa17095a-f2bf-4da6-8f08-ee6bfddb2063)

**Comment by JPH71** [comment]
Thanks for finding these...
Next time log them under issues, so i can track them quickly and then when
the fix is provided, you can confirm by testing and then getting the issue
closed.

I will tackle these soon...

Jason xxx

**Comment by dimmitri9** [comment]
Sorry! I totally forgot to do this instead :X

**Comment by JPH71** [comment]
Don't worry, I am the king of forgetfulness...

Trying to sort a number of issues around iafd...

I will incorporate the fixes for the errors you have found....

Cheers

Jason...

**Comment by JPH71** [comment]
Could you scrape these again - and send me the log files..... one log file per title!!!

**Comment by JPH71** [comment]
Also how did you name your file on disk??
I am getting different results from you..... but my cast is also not right,,,,
Dalton Riley, Say Uncle and SayUncle

**Reply by dimmitri9** [reply]
Is it fixed with the new code?

Goodnight Jason!

**Comment by JPH71** [comment]
I named my file: (Brother Crush) - Dalton Riley and Oliver Beck – Rocking His Boat (2023)

**Comment by dimmitri9** [comment]
[com.plexapp.agents.QueerClick.log](https://github.com/CodyBerenson/PGMA-Modernized/files/11672388/com.plexapp.agents.QueerClick.log)
Named: (Say Uncle) - Brother Crush - Dalton Riley and Oliver Beck – Rocking His Boat (2023)

**Comment by JPH71** [comment]
Not yet...
Another day or two...
Jason xxx

**Comment by JPH71** [comment]
Code is with Cody... thankyou once again for spotting this!

**Comment by dimmitri9** [comment]
Is it out? Goodnight!

**Comment by JPH71** [comment]
I just made an edit....
And sent it to Cody....
It should be up in the repository soon....
Previous update had an issue with the cast list....
